src/Components/RegisterToVote/RegisterToVote.js

```diff
@@ -0,0 +1,18 @@
+import './RegisterToVote.css';
+
+const RegisterToVote = () => {
+  return (
+    <div className="register-to-vote-form">
+      <iframe className="register-to-vote-i-frame"
+        src="https://register.vote.org/?partner=111111&campaign=free-tools" 
+        id="frame1"
+        scrollable="no"
+        scrolling="no"
+        title="register-to-vote-form"
+      />
+      <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/iframe-resizer/3.5.3/iframeResizer.min.js"></script>
```

Comment:
Interesting!  So did the two scripts included with this tool provided by vote.org do the same thing?  Just based on the readable text, it seems like the both gave the iFrame to ability to be resized.  I was wondering if they were both included to cover all of the browsers (*cough* IE *cough*).